### ISSUE TITLE
Linux: ship .desktop file with tarball

### ISSUE THREAD

**Issue by eszlari** [Microsoft/vscode#41294]
The deb/rpm files provide `/usr/share/applications/code.desktop`. Please also ship this file with the tarball (e.g. https://vscode-update.azurewebsites.net/1.19.1/linux-x64/stable), to make repackaging for us in flatpak (https://github.com/flathub/com.visualstudio.code) easier.

**Comment by Tyriar** [comment]
You could download the deb, extract and pull it from there as part of your build process. Might be better to move back to pulling the deb as it has this file, the URL is stable at https://vscode-update.azurewebsites.net/1.19.2/linux-deb-x64/stable

**Comment by eszlari** [comment]
> You could download the deb, extract and pull it from there as part of your build process.

That's actually what was done before. The switch to tarballs happened because nobody knew there was a stable deb URL. But including a .desktop file would also benefit users of the tarball, because they could link the file to `$HOME/.local/share/applications` to gain some desktop integration.

**Comment by Tyriar** [comment]
> The switch to tarballs happened because nobody knew there was a stable deb URL.

Yeah I suspected as much, I needed to do some digging to find the URL.

I'll leave this open, the first steps will involve converting the CI script here https://github.com/Microsoft/vscode/blob/a81a909dfea3b24f15df7d1991270021221246d3/build/tfs/linux/release.sh#L31-L34 into a gulp task in [gulpfile.vscode.linux.js](https://github.com/Microsoft/vscode/blob/master/build/gulpfile.vscode.linux.js), similar to how [the Windows zip is created](https://github.com/Microsoft/vscode/blob/a81a909dfea3b24f15df7d1991270021221246d3/build/gulpfile.vscode.win32.js#L78-L92).
